### PR TITLE
doc: nRF5340 audio fixes

### PR DESCRIPTION
**Pull request by greg-fer**
- doc: nrf5340_audio: remove child image mention

   Removed the mention of the child image directory from the docs.
   The application uses sysbuild as other applications.
   This mention was a doc bug.
   OCT-3244 and NCSDK-30393.

- doc: nrf5340_audio: clarify building steps

   Provided extra information for the building steps, including
   cross-links. Updated configuration file list. Added a section on the
   FOTA page about building. TECHDOC-3100.

### CODE REVIEW

File: applications/nrf5340_audio/doc/building.rst [hunk 1]
```diff
@@ -42,19 +42,27 @@ You can now program the development kit.
...
+
+.. literalinclude:: ../tools/buildprog/nrf5340_audio_dk_devices.json
+    :language: json
+
+When preparing the JSON file, update the following fields:
 
 * ``nrf5340_audio_dk_snr`` -- This field lists the SEGGER serial number.
-  You can check this number on the sticker on the nRF5340 Audio development kit.
-  Alternatively, connect the development kit to your PC and run ``nrfjprog -i`` in a command window to print the SEGGER serial number of the kit.
+  You can check this five-digit number on the sticker on the nRF5340 Audio development kit.
```
Comment (alexsven) [inline applications/nrf5340_audio/doc/building.rst]: The json-file needs the entire 10-digit number to flash. There is a 12-digit number when running nrfutil device list, so if the first 2 zeros are removed we have the correct 10-digit number

File: applications/nrf5340_audio/doc/building.rst [hunk 2]
```diff
@@ -65,107 +73,136 @@ Before using the script, make sure to update this file with the following inform
 Running the script
 ==================
 
-After editing the :file:`nrf5340_audio_dk_devices.json` file, run :file:`buildprog.py` to build the firmware for the development kits.
-The building command for running the script requires providing the following parameters:
-
-* Core type (``-c`` parameter): ``app``, ``net``, or ``both``
-* Application version (``-b`` parameter): either ``release`` or ``debug``
-* Device type (``-d`` parameter): ``headset``, ``gateway``, or ``both``
+The script can handle building and programming at the same time.
```
Comment (koffes) [inline applications/nrf5340_audio/doc/building.rst:76]:
```suggestion
The script handles building and parallel programming of multiple kits.
```

File: applications/nrf5340_audio/doc/building.rst [hunk 3]
```diff
@@ -65,107 +73,136 @@ Before using the script, make sure to update this file with the following inform
...
+Script parameters for programming
+---------------------------------
 
-   The following parameters are available for programming:
+The script can program the build files as part of the same `python buildprog.py` command used for building.
+Use one of the following programming parameters:
 
-   * Programming (``-p`` parameter) -- If you run the building script with this parameter, you can program one or both of the cores after building the files.
-   * Sequential programming (``-s`` parameter) -- If you encounter problems while programming, include this parameter alongside other parameters to program sequentially.
+* Programming (``-p`` parameter) -- If you run the building script with this parameter, you can program one or both of the cores after building the files.
```
Comment (koffes) [inline applications/nrf5340_audio/doc/building.rst:124]:
```suggestion
* Programming (``-p`` parameter) -- If you run the buildprog script with this parameter, you can program one or both of the cores after building the files.
```

File: applications/nrf5340_audio/doc/fota.rst [hunk 4]
```diff
@@ -68,9 +68,17 @@ The first part of these names is based on :kconfig:option:`CONFIG_BT_DEVICE_NAME
    This is due to restrictions in the DFU system, and the error print is expected.
    The DFU process should still complete successfully.
 
+Building the FOTA configuration
+*******************************
+
+Use the :ref:`nrf53_audio_app_building_standard` procedure to build the nRF5340 Audio applications with the FOTA configuration.
+Make sure to provide the :ref:`correct configuration file <nrf53_audio_app_building_config_files>` :file:`prj_fota.conf` when running the build command.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply applications/nrf5340_audio/doc/fota.rst:75]: Added in building.rst.

File: applications/nrf5340_audio/doc/fota.rst [hunk 5]
```diff
@@ -68,9 +68,17 @@ The first part of these names is based on :kconfig:option:`CONFIG_BT_DEVICE_NAME
...
+
+The :ref:`script-based method <nrf53_audio_app_building_script_running>` does not support building and programming the FOTA upgrades.
```
Comment (koffes) [inline applications/nrf5340_audio/doc/fota.rst:77]: Should we link to some pm_static.yaml description? Getting this set up right is important.

File: applications/nrf5340_audio/doc/building.rst [hunk 6]
```diff
@@ -246,19 +285,21 @@ After building the files for the development kit you want to program, follow the
...
+* Right headset (``--val 1``):
 
    .. code-block:: console
 
       nrfjprog --memwr 0x00FF80F4 --val 1
 
-Select the correct board when prompted with the popup or add the ``--snr`` parameter followed by the SEGGER serial number of the correct board at the end of the ``nrfjprog`` command.
+Select the correct board when prompted with the popup.
+Alternatively, you can add the ``--snr`` parameter followed by the SEGGER serial number of the correct board at the end of the ``nrfjprog`` command.
+You can check the serial numbers of the connected devices with the ``nrfutil device list`` command.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply applications/nrf5340_audio/doc/building.rst:302]: I've replaced nrfjprog for this reason in this PR, but I will mention both.

File: applications/nrf5340_audio/doc/requirements.rst [hunk 7]
```diff
@@ -60,6 +60,8 @@ For each application, only one of the following :file:`.conf` files is included
   No debug features are enabled in the release application version.
   When building using the command line, you must explicitly specify if :file:`prj_release.conf` is going to be included instead of :file:`prj.conf`.
   See :ref:`nrf53_audio_app_building` for details.
-
-In addition, the application features the :file:`child_image` directory with :file:`hci_ipc.conf`.
-This file contains the necessary configurations for nRF5340 Audio applications to run the :zephyr:code-sample:`bluetooth_hci_ipc` sample with :ref:`SoftDevice Controller for LE Isochronous Channels <nrfxlib:softdevice_controller_iso>` support.
+* :file:`prj_fota.conf` is the optional configuration file used for FOTA DFU.
+  When used, the build system builds the debug version of the application (:file:`prj.conf`), but with the features needed to perform DFU over Bluetooth LE.
```
Comment (koffes) [inline applications/nrf5340_audio/doc/requirements.rst:64]: Now I got a bit uncertain. Will this build the debug version?

File: applications/nrf5340_audio/doc/fota.rst (hunk 5 above)
Comment (greg-fer) [reply applications/nrf5340_audio/doc/fota.rst:77]: We have this caution earlier in the configuration section

> .. caution::
   The application is provided with a memory partition configuration in :file:`pm_static_fota.yml`, which is required to perform FOTA using an external flash.
   The partition configuration is an example that can be changed between versions, and can be incompatible with existing devices.
   For this reason, always create a partition configuration that suits your application.
   See :ref:`partition_manager` for more information on how to create a partition configuration.

File: applications/nrf5340_audio/doc/building.rst [hunk 8]
```diff
@@ -42,19 +42,27 @@ You can now program the development kit.
...
+
+.. literalinclude:: ../tools/buildprog/nrf5340_audio_dk_devices.json
+    :language: json
+
+When preparing the JSON file, update the following fields:
 
 * ``nrf5340_audio_dk_snr`` -- This field lists the SEGGER serial number.
-  You can check this number on the sticker on the nRF5340 Audio development kit.
-  Alternatively, connect the development kit to your PC and run ``nrfjprog -i`` in a command window to print the SEGGER serial number of the kit.
+  You can check this five- or ten-digit number on the sticker on the nRF5340 Audio development kit.
```
Comment (alexsven) [inline applications/nrf5340_audio/doc/building.rst:64]: It has to be ten-digit. I tested with five, doesn't work

**Review by alexsven — APPROVED** [review]

**Review by rick1082 — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**